### PR TITLE
[ANN-402][external] adding extract views flag in darwin py

### PR DESCRIPTION
**Pull request by tomvars**
Adds the option to add `--extract_views` at upload to get volumetric medical data split into its orthogonal planes.

### CODE REVIEW

File: darwin/dataset/upload_manager.py [hunk 1]
```diff
@@ -162,9 +162,19 @@ def serialize(self):
 
     def serialize_v2(self):
```
Comment (owencjones) [inline darwin/dataset/upload_manager.py:163]: It's not a change you've made, but this function `serialize_v2` doesn't actually seem to serialize/serialise data, just makes it ready for serialisation?

File: tests/darwin/cli_functions_test.py [hunk 2]
```diff
@@ -86,6 +87,7 @@ def with_verbose_flag(
                     None,
                     None,
                     False,
+                    False,
```
Comment (owencjones) [inline tests/darwin/cli_functions_test.py:90]: More a general comment for this line, but I'd consider annotating (not to confuse things here - not the imagery type!) these properties, e.g.

```
upload_data(
    argumentname=argumentvalue
    [repeated as needed]
)
```

Just for test readability.

Not necessarily for now.  Also looks like a test is needed at some point for the new extract flag you've added, but that these tests don't have them for others so not as part of this task.

**Review by owencjones — APPROVED** [review]
Looks good, and good to see some of the code.

I've added some comments, and only one (the typo) needs definite attention.  The others are purely stylistic, and techdebt/backlogable.

File: darwin/dataset/upload_manager.py [hunk 3]
```diff
@@ -162,9 +162,19 @@ def serialize(self):
...
         return {
-            "slots": [{"file_name": self.data["filename"], "slot_name": "0", "fps": self.data["fps"]}],
+            "slots": [
+                {
+                    "file_name": self.data["filename"],
+                    "slot_name": "0",
+                    "fps": self.data["fps"],
+                    "extract_views": self.data["extract_views"],
```
Comment (rslota) [inline darwin/dataset/upload_manager.py:170]: This change makes `extract_views` mandatory for SDK users, which it shouldn't. Please see how I refactored this here: https://github.com/v7labs/darwin-py/pull/486 and let's make this in similar way. I'm hoping my PRs will get merged soon, only waiting for QA now. Which yours will be waiting for too actually.

**Review by rslota — CHANGES_REQUESTED** [review]

File: darwin/dataset/upload_manager.py (hunk 3 above)
Comment (tomvars) [reply darwin/dataset/upload_manager.py]: Could you update this ? change has been made

**Pull request merged**